wolfcrypt/src/sp_arm64.c

```diff
@@ -20,6 +20,8 @@
  */
 
 /* Implementation by Sean Parkinson. */
+/* Thanks to Yair Poleg (yair.poleg@ayyeka.com) of Ayyeka for proposing static
```

Comment:
Please put the attribution once in the REAMDE for the forthcoming release notes.

Reply:
@SparkiDev : I think Todd would like this moved to the README. Once that change is made feel free to assign over to me and I can review and merge. Thanks